### DISCUSSION TITLE
Is it possible to import/merge projects?

### DISCUSSION THREAD

**Discussion by Akron** [vkbo/novelWriter#2116] [Q&A] [answered]
First of all: Many, many thanks for Novelwriter! I really enjoy using it!

But after writing a couple of short stories, I have found the following problem: I started writing some short stories separately, but now would like to join them in a single project, as they share a timeline and characters. Is it possible to import projects in another project - or merge some projects into one?

**Comment by vkbo** [comment, marked as answer]
Sort of. There is no direct way to merge, but there are two options.

* The clean way: Build a "novelWriter Markup" document from the Manuscript tool with the content you want imported, create a new document in the other project, and import or copy/paste the text there. Then you can use the split tool to split them up again into individual documents. (Right click the document in the project tree, select Transform and Split ...)
* The quick and dirty way: If you take all the files of the "content" folder under a novelWriter project and copy them to another one when they're both closed, the copied files will be imported into the project when you open it. They will be flagged as "[Recovered]" + the original title, so you'll have to rename and move them around.

I have been thinking of adding a way to import documents from another project. There is an import feature proposed in #1980, and also #2038. The latter is most relevant to your suggestion, so feel free to add to the proposal there.

**Comment by Akron** [comment]
Thank you very much - that is very helpful! I've tried both approaches, but I'm afraid that in my case the cleanup effort would be out of proportion to the benefit - I could probably live better with separate projects.
